filetracker/servers/storage.py

```diff
@@ -359,10 +359,8 @@ def _makedirs(path):
 
 def lutime(path, time):
     if six.PY2:
-        # In Python 2 utime always follows symlinks
         t = email.utils.formatdate(time)
-        if subprocess.call(['touch', '-c', '-h', '-d', t, path]) != 0:
+        if subprocess.call(["touch", "-c", "-h", "-d", t, path]) != 0:
```

Comment:
Why is this? We usually have untranslatable strings in single quotes?

Reply:
Sorry, this is a result of some chaos ensuing because of working on multiple feature branches at once and applying changes to the wrong branch.